src/sentry/static/sentry/app/components/demo/demoHeader.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import createReactClass from 'create-react-class';
+import Reflux from 'reflux';
+
+import sentry from 'sentry-images/logos/logo-sentry.svg';
+
+import Button from 'app/components/button';
+import ButtonBar from 'app/components/buttonBar';
+import ExternalLink from 'app/components/links/externalLink';
+import {t} from 'app/locale';
+import OrganizationStore from 'app/stores/organizationStore';
+import {Organization} from 'app/types';
+import {trackAdvancedAnalyticsEvent} from 'app/utils/advancedAnalytics';
+
+type Props = {organization?: Organization};
+
+function DemoHeader({organization}: Props) {
+  return (
+    <Wrapper>
+      <ImageAndName>
+        <Image />
+        <SentryWrapper>Sentry</SentryWrapper>
+      </ImageAndName>
+      <StyledButtonBar gap={1}>
+        <StyledExternalLink href="https://docs.sentry.io">
+          {t('Documentation')}
+        </StyledExternalLink>
+        <GetStarted
+          onClick={() =>
+            trackAdvancedAnalyticsEvent('growth.demo_click_get_started', {}, organization)
```

Comment:
We don't really need org here do we? Can we just pass some mock org and kill the reflux stuff?

Reply:
@EvanPurkhiser Eh, I think it could be worse in the long run if we passed in a dummy org as that could make some analysis of analytics harder. The reflux stuff is shitty but it's not _that_ bad and could easily be ripped out if we needed to ax it.